Sources/Rendering/Core/Renderer/index.d.ts

```diff
@@ -35,6 +35,9 @@ export interface IRendererInitialValues extends IViewportInitialValues {
 	occlusionRatio?: number;
 	maximumNumberOfPeels?: number;
 	texturedBackground?: boolean;
+	backgroundTextureDiffuseStrength?: number;
+	backgroundTextureSpecularStrength?: number;
+	useTextureAsViewBackground?: boolean;
```

Comment:
Thinking about this some more what do you think of
- environmentTexture
- environmentTextureDiffuseStrength
- environmentTextureSpecularStrength
- useEnvironmentTextureAsBackground

On the C++ side we support using a texture map as a static background image and I'm thinking this may help clarify the difference.

Reply:
that would make more sense to me, but I didn't change it because the renderer already had a backgroundTexture property and I didn't want to change the names of any values that people may already be using. If renaming it is okay then I'm all for it

Reply:
Yeh, let''s use EnvironmentTexture that is what it is called in C++, backgroundTexture is a static 2D wallpaper type texture .